config/services_tests_config_prod.json

```diff
@@ -0,0 +1,21 @@
+{
+    "all": "harmony,harmony-regression,geoloco,hga,n2z,regridder,hoss,hybig,net2cog,subset-band-name,swath-projector,nsidc-icesat2,trajectory-subsetter",
+    "batchee": "TBD",
```

Comment:
Do these TBDs cause issues when triggered? I just want to make sure we don't see failures in the UI when there are no tests to trigger.

Reply:
github is smart enough to just ignore TBDs. It is not listing them in the workflow dispatch options and when triggered by repository dispatch, it just return a 204 with no tests run.